exciting_environments/cart_pole/cart_pole_env.py

```diff
@@ -212,18 +211,30 @@ def vector_field(t, y, args):
         omega_k1 = y[3]
         theta_k1 = ((theta_k1 + jnp.pi) % (2 * jnp.pi)) - jnp.pi
 
-        phys = self.PhysicalState(
-            deflection=deflection_k1,
-            velocity=velocity_k1,
-            theta=theta_k1,
-            omega=omega_k1,
-        )
-        opt = None  # Optional(something=...)
-        return self.State(physical_state=phys, PRNGKey=None, optional=opt)
+        with jdc.copy_and_mutate(state, validate=False) as new_state:
```

Comment:
Should we investigate why `validate=False` is necessary here? I think it generally should not need to be, looking at the documentation for `jdc`

Reply:
Seems to have caused problems in an older version. The validation is now activated again and works without any problems.